### PR TITLE
Create pedal feature

### PR DESCRIPTION
**Pull request by matheusmigge**

### CODE REVIEW

File: PedalCore/HomeView.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ public struct HomeView: View {
         
     }
     public var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        Text("List of pedals")
     }
```
Comment (matheusberger) [inline PedalCore/HomeView.swift]: Nice

Comment (matheusmigge) [reply PedalCore/HomeView.swift]: Very nice.

File: PedalCore/ViewModels/HomeViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class HomeViewModel: ObservableObject {
+    @Published var pedals: [Pedal] = []
+
+    func addPedal(name: String, brand: String, knobNames: [String]) {
+        let knobs = Dictionary(uniqueKeysWithValues: knobNames.map { name in
+            (name, 0)
+        })
+        let newPedal = Pedal(name: name, brand: brand, knobs: knobs)
+        pedals.append(newPedal)
+        print(pedals)
```
Comment (matheusberger) [inline PedalCore/ViewModels/HomeViewModel.swift]: Remember to remove any print() code you used before the final PR review
```suggestion
```

**Review by matheusberger — CHANGES_REQUESTED** [review]
Please remove the print in HomeViewModel the re-request my review.

**Review by matheusberger — APPROVED** [review]

**Pull request merged**